### PR TITLE
Add Room dependency and LoggedInUser class

### PR DESCRIPTION
**Pull request by tiembo**

### CODE REVIEW

File: core/build.gradle [hunk 1]
```diff
@@ -87,6 +87,8 @@ dependencies {
     api "org.jsoup:jsoup:${versions.jsoup}"
     api "android.arch.lifecycle:viewmodel:${versions.lifecycle}"
     api "android.arch.lifecycle:extensions:${versions.lifecycle}"
+    api "android.arch.persistence.room:runtime:${versions.room}"
```
Comment (keyboardsurfer) [inline core/build.gradle]: Please sort into existing dependencies alphabetically.

File: core/src/main/java/io/plaidapp/core/designernews/data/users/model/LoggedInUser.kt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    val lastName: String,
+
+    @ColumnInfo(name = "display_name")
+    @SerializedName("display_name")
+    val displayName: String,
+
+    @ColumnInfo(name = "potrait_url")
+    @SerializedName("portrait_url")
+    val portraitUrl: String? = null
+)
```
Comment (keyboardsurfer) [inline core/src/main/java/io/plaidapp/core/designernews/data/users/model/LoggedInUser.kt]: Mandatory missing new line nit. (Can be enabled directly in Android Studio)

File: build.gradle [hunk 3]
```diff
@@ -42,7 +42,8 @@ buildscript {
             'crashlytics'       : '2.9.4',
             'googleServices'    : '4.0.1',
             'fabric'            : '1.25.4',
-            'firebase'          : '16.0.1'
+            'firebase'          : '16.0.1',
+            'room'              : '1.1.1'
```
Comment (keyboardsurfer) [inline build.gradle]: Please also sort this one in alphabetically.

File: core/src/main/java/io/plaidapp/core/designernews/data/users/model/LoggedInUser.kt [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+package io.plaidapp.core.designernews.data.users.model
+
+import android.arch.persistence.room.ColumnInfo
+import android.arch.persistence.room.Entity
+import android.arch.persistence.room.PrimaryKey
+import com.google.gson.annotations.SerializedName
+
+@Entity(tableName = "logged_in_user")
+data class LoggedInUser(
+    @PrimaryKey
```
Comment (florina-muntenescu) [inline core/src/main/java/io/plaidapp/core/designernews/data/users/model/LoggedInUser.kt]: can you put these on one line?

File: core/src/main/java/io/plaidapp/core/designernews/data/users/model/LoggedInUser.kt (hunk 2 above)
Comment (tiembo) [reply core/src/main/java/io/plaidapp/core/designernews/data/users/model/LoggedInUser.kt]: Done

**Pull request merged**